tests/test_main.py

```diff
@@ -86,6 +86,20 @@ def test_prog():
             build.__main__.main(['--help'], prog='something')
 
 
+def test_version():
+    out = io.StringIO()
```

Comment:
No need for this magic. Just let the version go through and use the capsys fixture from pytest.